src/main/thrift/parquet.thrift

```diff
@@ -190,6 +190,45 @@ enum FieldRepetitionType {
   /** The field is repeated and can contain 0 or more values */
   REPEATED = 2;
 }
+/**
+ * A structure for capturing metadata for estimating the unencoded, uncompressed size
+ * of data.
+ *
+ * Writers should populate all fields in this struct except for the exceptions listed per field.
+ */ 
+struct SizeEstimationStatistics {
+   /** 
+    * The number of logical physical bytes stored for BYTE_ARRAY data values. Logical bytes refers to the number
+    * of bytes needed if no special encoding is used. This is exclusive of the bytes needed
+    * to store the length of each byte array. In other words, this field is equivelant to the the (size of 
+    * PLAIN-ENCODING the byte array values) - (4 bytes * number of values written). To determine logical sizes 
```

Comment:
4 bytes * number of values written
is that the logical size of the data? Is this assuming that each value takes 4 bytes of space?

Reply:
plain encoding of bytes arrays writing all strings as `(4 bytes for length of data)(actual data)` back to back.